src/api/app/views/webui/request/beta_show.html.haml

```diff
@@ -26,33 +26,50 @@
 
 .card
   .card-body.p-0
-    .card-title.px-4.pt-4.mb-0
+    .card-title.p-4.mb-0
       %h3
         Request #{@bs_request.number}
         %span.badge.ml-1{ class: "badge-#{request_badge_color(@bs_request.state)}" }
           = @bs_request.state
           - if @bs_request.superseded_by.present?
             by
             = link_to(@bs_request.superseded_by, number: @bs_request.superseded_by)
-    %p.font-italic.px-4
-      Created by
-      = user_with_realname_and_icon(@bs_request.creator)
-      = fuzzy_time(@bs_request.created_at)
-      - if @bs_request.superseding.present?
-        = '. Supersedes'
-        - @bs_request.superseding.each do |supersed|
-          = link_to "##{supersed['number']}", number: supersed['number']
-    %p.px-4= request_action_header(@actions.first, @bs_request.creator) # TODO: adapt when handling requests with multiple actions
+      %p.font-italic
+        Created by
+        = user_with_realname_and_icon(@bs_request.creator)
+        = fuzzy_time(@bs_request.created_at)
+        - if @bs_request.superseding.present?
+          = '. Supersedes'
+          - @bs_request.superseding.each do |supersed|
+            = link_to "##{supersed['number']}", number: supersed['number']
+
+      - if @bs_request.bs_request_actions.count > 1
+        %p This request contains multiple actions. Select one:
```

Comment:
The text `Select one:` is superfluous here, since the dropdown's text has the same call to action with `Select action`. We can remove `Select one:`.

Reply:
I've changed the text. Now it shows the same value as it is available in the dropdown menu

Reply:
@rubhanazeem I think the idea is to remote "Select one:" from the text before the dropdown. Not from the dropdown.